Cargo.lock

```diff
@@ -2348,8 +2348,10 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8d3b63360ec3cb337817c2dbd47ab4a0f170d285d8e5a2064600f3def1402397"
 dependencies = [
  "compiler_builtins",
+ "flate2",
  "rustc-std-workspace-alloc",
  "rustc-std-workspace-core",
+ "wasmparser",
```

Comment:
Hm, given that the other user of this crate is `libstd` (via `gimli`), won't this cause `wasmparser` (and `flate2`) to be compiled into `libstd` too?

Reply:
I think I forgot to update `Cargo.lock`. I disabled the wasm feature.